src/install/repository.zig

```diff
@@ -168,17 +191,69 @@ pub const Repository = extern struct {
             });
 
         switch (result.term) {
-            .Exited => |sig| if (sig == 0) return result.stdout,
+            .Exited => |sig| if (sig == 0) return result.stdout else if (
+                // remote: The page could not be found <-- for non git
+                // remote: Repository not found. <-- for git
```

Comment:
Another variation we can check for:
```
git clone invalidurl
fatal: repository 'invalidurl' does not exist
```

Reply:
although, I don't think that will make it through dependency parsing as a git dependency, so it's probably fine